pages/platform/kubernetes-k8s/backup-and-restore-cluster-namespace-and-applications-with-trilio/guide.en-us.md

```diff
@@ -0,0 +1,1278 @@
+  type: Full
+  clusterBackupPlan:
+    name: ovh-multi-ns-backup-plan
+    namespace: default
+```
+
+Steps to initiate a backup for all important namespaces in your OVH Managed Kubernetes Cluster:
+
+1. First, change directory where the `ovh/docs` Git repository was cloned on your local machine:
+
+    	```shell
+	cd docs
+	```
+
+2. Then, open and inspect the `ClusterBackupPlan` and `ClusterBackup` manifest files provided in the `docs` repository.
+
+	```shell
+    	cat assets/manifests/multi-ns-backup-plan.yaml
+    	cat assets/manifests/multi-ns-backup.yaml
+    	```
+
+3. Create the `ClusterBackupPlan` resource, using `kubectl`:
+
+    	```shell
+    	kubectl apply -f assets/manifests/multi-ns-backup-plan.yaml
+    	```
+
+Now, inspect the `ClusterBackupPlan` status, using `kubectl`:
+
+	```shell
+	kubectl get clusterbackupplan multi-ns-backup-plan -n default
+	```
+
+The output looks similar to (notice the `STATUS` column value which should be set to `Available`):
+
+	```text
+	NAME                            TARGET                 ...		STATUS
+	ovh-multi-ns-backup-plan		trilio-ovh-s3-target   ...		Available
+	```
+
+4. Finally, create the `ClusterBackup` resource, using `kubectl`:
+	
+	```shell
+	kubectl apply -f assets/manifests/multi-ns-cluster-backup.yaml
+	```
+
+Next, check the `ClusterBackup` status, using `kubectl`:
+
+	```shell
+	kubectl get clusterbackup multi-ns-cluster-backup -n default
+	```
+
+The output looks similar to (notice the `STATUS` column value which should be set to `Available`, as well as the `PERCENTAGE COMPLETE` set to `100`):
+
+	```text
+	NAME           		BACKUPPLAN             		BACKUP TYPE   STATUS		...		COMPLETE
+	multi-ns-backup   	ovh-multi-ns-backup-plan	Full          Avilable		...		100                               
+	```
+
+If the output looks like above then all your important application namespaces were backed up successfully.
+
+**Note:**  Please bear in mind that it may take a while for the full cluster backup to finish, depending on how many namespaces and associated resources are involved in the process.
+
+You can also open the web console main dashboard and inspect the `multi-namespace` backup (notice how all the important namespaces that were backed up are highlighted in green color, in a honeycomb structure)
+
+### Re-creating the `OVH Managed Kubernetes Cluster` and Restoring Applications
+
+An important aspect to keep in mind is that whenever you destroy a `OVH Managed Kubernetes Cluster` and then restore it, a new `Load Balancer` with a new external `IP` is created as well when `TVK` restores your `ingress` controller. So, please make sure to update your OVH Managed DNS `A records` accordingly.
+
+Now, delete the whole `OVH Managed Kubernetes Cluster` using the OVH Cloud Web Console (https://ca.ovh.com/manager/public-cloud)
+
+Next, re-create the cluster as described in [Creating a OVH Managed Kubernetes Cluster] (https://docs.ovh.com/gb/en/kubernetes/creating-a-cluster/#instructions).
```

Comment:
Please fix the link format here

Reply:
Thank you @OVH-Team-Guides ! I have fixed this.